tools/downloader/converter.py

```diff
@@ -160,10 +160,11 @@ def convert(reporter, model):
             for arg in model.mo_args]
 
         for model_precision in sorted(model_precisions):
+            dir_precision = model_precision + '-INT8' if model.quantized else model_precision
```

Comment:
This is not the right place for this. `model_precision` should already equal `FP32-INT8`/`FP16-INT8`. What you need to do is change the code in `common.py` that sets `model.precisions` so that it works appropriately for quantized models. That way:

* info dumper's output will be correct;
* Converter options like `--precisions=FP32-INT8` will work correctly.

Reply:
model_precision is also used as MO parameter '--data_type={}'.format(model_precision), and have to be FP32/16, other variants MO do not accept

Reply:
You can strip off the `-INT8` part when running MO.

Reply:
what is the difference between adding `-INT8` to folder name and stripping it out of MO parameter?

Reply:
The difference is that the latter preserves the property that a converted model is placed in the subdirectory with the name of the target precision. Running the converter with `--precision=FP32` should place the model in the `FP32` subdirectory, while running it with `--precision=FP32-INT8` should place it into `FP32-INT8`.

Reply:
ok, changed